src/main/java/com/notably/logic/LogicManager.java

```diff
@@ -29,28 +29,25 @@ public LogicManager(Model model, Storage storage) {
         this.model = model;
         this.storage = storage;
         notablyParser = new NotablyParser();
-
     }
 
     @Override
     public void execute(String commandText) throws CommandException, ParseException {
         logger.info("----------------[USER COMMAND][" + commandText + "]");
-        try {
-            storage.saveAddressBook(model.getAddressBook());
-        } catch (IOException ioe) {
-            throw new CommandException(FILE_OPS_ERROR_MESSAGE + ioe, ioe);
-        }
-    }
-
-    @Override
-    public ReadOnlyAddressBook getAddressBook() {
-        return model.getAddressBook();
-    }
 
-    @Override
-    public void getFilteredPersonList() {
```

Comment:
You can also remove these two methods from the Logic interface, as they are no longer in use.